whois-nrtm4/src/main/java/net/ripe/db/nrtm4/SnapshotFileGenerator.java

```diff
@@ -8,6 +8,7 @@
 import net.ripe.db.nrtm4.domain.NrtmSourceModel;
```

Comment:
you are not using the output of createSnapshots anywhere 
createSnapshotsForVersions visibility could be private (you are just using this method in this class)